From: Boyuan Yang <byang@debian.org>
Date: Sun, 5 Jan 2020 14:40:39 -0500
Subject: no git

Don't include git info in version.

Strip unavailable git information.
---
 main.c      | 4 ----
 makefile.in | 5 +----
 2 files changed, 1 insertion(+), 8 deletions(-)

diff --git a/main.c b/main.c
index 538ff2d..02a3de9 100644
--- a/main.c
+++ b/main.c
@@ -269,14 +269,10 @@ int main(int argc, char *argv[]) {
       if (1 || isatty(1))   // fix this.  for interactive from menu bar.
 #endif
       {
-#ifdef HAVE_WX
-	extern char *GIT;
-#endif
 	char version[64];
 	lcleartext(NIL);
 #ifdef HAVE_WX
 	strcpy(version,"6.1");
-	strcat(version,GIT);
 #else
 	strcpy(version,"5.6");
 #endif
diff --git a/makefile.in b/makefile.in
index 5ff8d61..78d011d 100644
--- a/makefile.in
+++ b/makefile.in
@@ -13,7 +13,7 @@ LINKER = @LINKER@
 
 OBJS 	= coms.o error.o eval.o files.o graphics.o init.o intern.o \
 	  libloc.o lists.o logodata.o main.o math.o mem.o paren.o parse.o \
-	  print.o wrksp.o nographics.o git.o obj.o @WXOFILES@ @TERMOFILE@ @GRAPHICSOFILE@
+	  print.o wrksp.o nographics.o obj.o @WXOFILES@ @TERMOFILE@ @GRAPHICSOFILE@
 
 SRCS	= coms.c error.c eval.c files.c graphics.c init.c intern.c \
 	  libloc.c lists.c logodata.c main.c math.c mem.c paren.c parse.c \
@@ -30,9 +30,6 @@ all:	ucblogo logolib/Messages helpfiles helpfiles/HELPCONTENTS
 mem.o:	mem.c
 	$(CC) $(CFLAGS) -O0 -c mem.c
 
-git.c:	$(SRCS)
-	echo 'char* GIT = "('`git describe||echo NA|tr -d '\r'`')";' > git.c
-
 tags:	$(SRCS)
 	ctags --format=1 -N $(SRCS) $(HDRS)
 # 	ctags -t $(SRCS) $(HDRS)
